core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionIntegrationSuite.scala

```diff
@@ -418,8 +418,11 @@ class BlockManagerDecommissionIntegrationSuite extends SparkFunSuite with LocalS
       .map(_.asInstanceOf[ShuffleIndexBlockId].shuffleId)
       .get
 
+    eventually(timeout(1.minute), interval(10.milliseconds)) {
+      val newShuffleFiles = shuffleFiles.diff(existingShuffleFiles)
+      assert(newShuffleFiles.size >= shuffleBlockUpdates.size)
+    }
     val newShuffleFiles = shuffleFiles.diff(existingShuffleFiles)
```

Comment:
After passing `eventually` in the above, we make this variable again to reuse in the next steps.